tensorboard/webapp/widgets/line_chart_v2/types.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// Only selectively export types that would be used by the user of the line chart.
+export {
+  DataSeries,
+  RendererType,
+  ScaleType,
+  DataSeriesMetadataMap,
+  DataSeriesMetadata,
+} from './lib/public_types';
```

Comment:
Thanks for the explicit export! Much easier to read these when poking
around the codebase than the `export *`s that we use in some places,
especially when there are multiple `export *`s in the same module.